### PR TITLE
Add proto support to logviewer/

### PR DESCRIPTION
**Pull request by mborsz**

### CODE REVIEW

File: logviewer/example/protos/test.proto [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+option go_package = "protos";
+
+message TestMessage {
+    google.protobuf.Timestamp test = 1;
+    k8s.io.perftests.logviewer.example.protos2.TestMessage2 nested = 2;
+}
+
+service TestService {
+    rpc Test(TestMessage) returns (stream TestMessage) {}
+}
```
Comment (mm4tt) [inline logviewer/example/protos/test.proto]: NIT: newline

File: logviewer/example/protos2/test2.proto [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+import "google/protobuf/empty.proto";
+
+package k8s.io.perftests.logviewer.example.protos2;
+
+option go_package = "protos2";
+
+message TestMessage2 {
+    google.protobuf.Empty empty = 1;
+}
```
Comment (mm4tt) [inline logviewer/example/protos2/test2.proto]: NIT: Newline

Comment (mborsz) [reply logviewer/example/protos2/test2.proto]: DONE

File: logviewer/example/protos/test.proto (hunk 1 above)
Comment (mborsz) [reply logviewer/example/protos/test.proto]: DONE

**Comment by mborsz** [comment]
Done

**Pull request merged**